### DISCUSSION TITLE
Можно ли использовать tplink tl-841n для обхода блокировок?

### DISCUSSION THREAD

**Discussion by hkyrylo** [bol-van/zapret#241] [Q&A]
имею на руках самый популярный стандартный роутер на 4/32. После установки openwrt остается что то 100 килобайт памяти.Можно ли что то сделать, установить в оперативку или что то такое, чтобы включить программу для обхода?

**Comment by bol-van** [comment]
можно только tpws с ручными  iptables
все остальное не влезет

**Comment by kotsmotritnastul** [comment]
судя по этой статье https://openwrt.org/supported_devices/432_warning такие роутеры постоянно крашатся, греются, тормозят и всё на них работает кое-как.

и кроме того насколько я знаю для PPPoE и L2TP нужно устанавливать отдельные пакеты, и если провайдер даёт именно такой интернет то не факт что они влезут в оставшееся пространство.

но можно попробовать использовать флешку как носитель для опенврт по такой инструкции https://openwrt.org/docs/guide-user/additional-software/extroot_configuration но я не знаю есть ли на этом роутере разъём для usb.
